requirements.txt

```diff
@@ -1,3 +0,0 @@
-requests
-feedparser
-Pillow
```

Comment:
Ok I will have the setup.py just read the lines in that file and generate the requirements.